x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.tsx

```diff
@@ -41,6 +41,7 @@ interface AlertsCountPanelProps {
   title?: React.ReactNode;
   isExpanded: boolean;
   setIsExpanded: (status: boolean) => void;
+  signalIndexName: string | null;
```

Comment:
Let's add a code documentation here as well. [This](https://github.com/elastic/kibana/blob/main/x-pack/solutions/security/plugins/security_solution/public/detections/components/alerts_kpis/alerts_summary_charts_panel/types.ts#L39) is what used for the other charts